monkey/monkey_island/cc/ui/src/services/AuthService.js

```diff
@@ -29,16 +33,47 @@ export default class AuthService {
       .then(response => response.json())
       .then(response => {
         if(response.meta.code === 200){
-          this._removeToken();
+          this._removeToken(this.AUTH_TOKEN_NAME_IN_LOCALSTORAGE);
         }
         return response;
       });
   }
 
   authFetch = (url, options) => {
+    this._refreshAuthToken(url, options);
```

Comment:
We're refreshing the token on every request?

Reply:
A mistake which will be fixed.